kohese-portal/client/src/components/type-editor/attribute-editor/attribute-editor.component.ts

```diff
@@ -15,17 +15,19 @@
  */
 
 
-import { Component, ChangeDetectionStrategy, ChangeDetectorRef, OnInit,
-  Input } from '@angular/core';
+// Angular
+import { Component, ChangeDetectionStrategy, ChangeDetectorRef, OnInit, Input } from '@angular/core';
```

Comment:
See previous comments about ChangeDetectionStrategy.  Investigate if it is actually needed.

Reply:
ChangeDetectionStrategy is used to change the detection strategy to OnPush.